### DISCUSSION TITLE
The list of songs in a playlist

### DISCUSSION THREAD

**Discussion by ehsaneona** [AzuraCast/AzuraCast#6313] [Q&A]
Hello to all.

I wanted to know if there is a way to get the songs of a Playlist? The current playlist request only gives the data of the playlist itself, I also want the data of the songs in the playlist.

Thank you for your guidance.

**Comment by BusterNeece** [comment]
Using the API, do a file list with the search phrase "playlist:playlist_name" to get all media with that playlist.
